### PR TITLE
Add sequence diagram for add_app

### PR DESCRIPTION
**Pull request by 0liverkong**

### CODE REVIEW

File: docs/diagrams/TagSequenceDiagram.puml
Comment (acekhoon) [inline docs/diagrams/TagSequenceDiagram.puml:1]: This one can wait until Wei Ming's PR is merged

**Review by acekhoon — COMMENTED** [review]
I think your image is not updated yet so that it fixes the error from your original UML diagram. Please update the image accordingly.

**Review by leeweiming3 — APPROVED** [review]
LGTM.

**Review by acekhoon — APPROVED** [review]
LGTM

**Pull request merged**
